diff --git a/src/core/cli/config.rs b/src/core/cli/config.rs
index 159affc..0dc32e7 100644
--- a/src/core/cli/config.rs
+++ b/src/core/cli/config.rs
@@ -12,7 +12,7 @@ impl Default for Config {
             home::home_dir().expect("Failed to retrieve home directory"),
         )
         .expect("Invalid home directory");
-        let lurk_dir = home_dir.join(".lurk");
+        let lurk_dir = home_dir.join(".local").join("share").join("lurk");
         Self { lurk_dir }
     }
 }
